src/Makevars.in

```diff
@@ -1,5 +1,7 @@
-PKG_CFLAGS = @PKG_CFLAGS@ @openmp_cflags@ @zlib_cflags@
-PKG_LIBS = @PKG_LIBS@ @openmp_cflags@ @zlib_libs@
+PKG_CFLAGS=-g -O3 -flto=auto -fno-common -fopenmp -Wall -Wvla -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2
+PKG_CXXFLAGS=-g -O3 -flto=auto -fno-common -fopenmp -Wall -Wvla -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2
```

Comment:
don't think we can put Wvla here given the issue with it not being portable

Reply:
Good catch. Makevars slipped while trying to reproduce other CI error :(